foundry.toml

```diff
@@ -17,12 +20,24 @@ runs=500
 fail_on_revert = true
 depth = 10
 
-[profile.lite]
-solc = '0.8.19'
-via_ir = false
-optimizer = true
-optimizer_runs = 10_000
-ignored_error_codes = [3628]
+[profile.optimized-build]
+via_ir = true
+test = 'src'
+out = 'out-optimized'
+bytecode_hash = 'none'
```

Comment:
There seems to be some conflicting information on the current state of the config field `bytecode_hash`, but I think we should prefer to remove this line and use the default value.

According to the Foundry Book, if this is set to none, source code verification on some block explorers becomes impossible: https://book.getfoundry.sh/forge/deploying?highlight=bytecode_hash#verifying-contracts-with-no-bytecode-hash

However, more recent sources indicate that 'none' is actually the default value, and that verification is fixed to support this: https://github.com/foundry-rs/foundry/issues/1236